spec/spec_helper.rb

```diff
@@ -4,7 +4,16 @@
 # https://voxpupuli.org/docs/updating-files-managed-with-modulesync/
 
 RSpec.configure do |c|
-  c.mock_with :mocha
+  c.before do
+    # select the systemd service provider even when on docker
+    # https://tickets.puppetlabs.com/browse/PUP-11167
+    if defined?(facts) and  %w[Archlinux RedHat].include?(facts[:os]['family'])
```

Comment:
why limit to to those two OSes? shouldn't we do it on all?

Reply:
Was thinking someone might add slackware at some future date and then you might get the wrong service provider.
Archlinux, RedHat is everything in metadata.yml today so for sure not against changing.